docs/src/main/asciidoc/_elytron/Elytron_Subsystem.adoc

```diff
@@ -1011,10 +911,10 @@ realm.
             "path" => "mgmt-groups.properties",
             "relative-to" => "jboss.server.config.dir"
         },
-        "plain-text" => false,
         "users-properties" => {
```

Comment:
Strange, there is plain-text attribute in users-properties. Just is not listed by :read-resource() operation

Reply:
May need include-defaults=true on the read call

Reply:
have already tried that - no luck 
```
[standalone@localhost:9990 /] /subsystem=elytron/properties-realm=ManagementRealm:read-resource(include-defaults=true)
{
    "outcome" => "success",
    "result" => {
        "groups-attribute" => "groups",
        "groups-properties" => {
            "path" => "mgmt-groups.properties",
            "relative-to" => "jboss.server.config.dir"
        },
        "users-properties" => {
            "path" => "mgmt-users.properties",
            "relative-to" => "jboss.server.config.dir",
            "digest-realm-name" => "ManagementRealm"
        }
    }
}
```

Reply:
Added plain-text attribute back

Reply:
Bug tracked by WFCORE-4041